pkgs/development/python-modules/py-radix-sr/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, coverage
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "py-radix-sr";
+  version = "1.0.0.post1";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-4MD5IjgIVrvfeFxwH2dmHxtcXLZ3kwhTLOPCemIEzX0=";
+  };
+
+  pythonImportsCheck = [
+    "radix"
+  ];
+
+  nativeCheckInputs = [ coverage nose ];
```

```suggestion
  nativeCheckInputs = [
    pytestCheckHook
  ];
```
We're not interested in coverage and need something that actually runs the tests.